config-application-package/src/test/java/com/yahoo/config/application/HostedOverrideProcessorComplexTest.java

```diff
@@ -52,8 +51,8 @@ public void testProdBetaUsWest2a() throws TransformerException {
                         <redundancy>1</redundancy>
                     </content>
                     <content id="filedocument" version="1.0">
-                        <nodes count="2" groups="2">
-                            <resources disk="32Gb" memory="8Gb" vcpu="4"/>
+                        <nodes count="2" groups="2" required="true">
```

Comment:
Why `required="true"` all of a sudden?

Reply:
It is special cased for the node element in https://github.com/vespa-engine/vespa/blob/master/config-application-package/src/main/java/com/yahoo/config/application/OverrideProcessor.java#L230-L238
I don't know exactly why this is done, but it would not happen if I had tested the override on another element and I'm pretty sure this PR doesn't change the behaviour, only makes it happen in the unit test.

Reply:
Thanks, that makes perfect sense.